src/components/root.tsx

```diff
@@ -36,25 +35,22 @@ const Root = () => {
 
     return (
         <ItemsContextProvider>
-            <div
-                className={`theme-${theme} h-screen text-text`}
-                data-theme={'ph-' + theme}
-            >
+            <div className="h-screen">
                 {isElectron && (
                     <div
-                        className={`h-[30px] w-full
+                        className={`h-8 w-full
                     ${isMac && 'justify-center'}
-                    z-90 flex items-center bg-titlebar
+                    flex items-center bg-gray-50
                     [-webkit-app-region:drag]`}
                     >
-                        <IconLogo className="ml-1 mr-[2px] p-[6px]" />
+                        <IconLogo className="ml-1 mr-0.5 p-1.5" />
                         <IconConnect className="h-3 w-fit" />
                     </div>
                 )}
                 <div
                     className={`flex items-stretch overflow-auto
-                        ${isElectron ? 'h-[calc(100vh-30px)]' : 'h-screen'}
-                        ${isDropTarget ? 'bg-light' : 'bg-bg'}
+                        ${isElectron ? 'h-app-height' : 'h-screen'}
```

Comment:
over time I will try to extract re-usable spacing values like this `app-height` so that they are easier to use